### PR TITLE
add "id" property of form

### PR DESCRIPTION
**Pull request by rodryquintero**
This is needed to be able to assign submit buttons that are outside the
form tags.

### CODE REVIEW

File: src/main.js [hunk 1]
```diff
@@ -435,7 +435,7 @@ Formsy.Form = React.createClass({
   render: function () {
 
     return (
-      <form {...this.props} onSubmit={this.submit}>
+      <form id={this.props.id} {...this.props} onSubmit={this.submit}>
```
Comment (Semigradsky) [inline src/main.js]: It is not necessary. `{...this.props}` does it

**Comment by Semigradsky** [comment]
Already, you can use this:
```js
<Formsy.Form id="some-id"></Formsy.Form>
```

File: src/main.js (hunk 1 above)
Comment (rodryquintero) [inline src/main.js]: I thought so, however, the version I downloaded for use in one of my projects does not. Any ideas why?

Comment (rodryquintero) [inline src/main.js]: I just noticed that the npm version downloaded is different from the version in the repository. That version does not have {...props}. Just so you know.

Comment (Semigradsky) [inline src/main.js]: It seems that it was not included in the release.
You can use the latest version in this way:
```
npm install --save christianalfoni/formsy-react
```

@christianalfoni how about to create new 0.14.x version before 0.15.0 ?

Comment (christianalfoni) [inline src/main.js]: If we just hold our breaths a little bit more I think React 0.14 is right around the corner :-) What we could do though is update the current readme encouraging people to use the new branch with React 0.14 Beta. That helps Facebook testing new React and us testing new Formsy.

I think most issues now can be worked around until next release.

**Pull request closed without merging**